tupaia-packages.code-workspace

```diff
@@ -95,6 +95,7 @@
       "Codeship",
       "DHIS",
       "MediTrak",
+      "PSSS",
```

Comment:
Just a warning, I think @edmofro has also done this in: https://github.com/beyondessential/tupaia/pull/1686/files#diff-65619f8f662c566b8a93965b22b0b01a7aa4668821d23dd9e727b975ed5f6291R102

Reply:
Thanks for flagging this! Hopefully the conflict would be resolved automatically, but if not it should be easy to resolve it